### DISCUSSION TITLE
[blocks]:video block component that could serve as a template for video lessons

### DISCUSSION THREAD

**Discussion by vksssd** [shadcn-ui/ui#3852] [Blocks Request]
### Description

video block component that could serve as a template for video lessons.
This component would ideally include features such as:

Video player area
Course title and description sections
Instructor information section
Progress tracking (e.g., progress bar, completed/remaining lessons)
Navigation controls (e.g., next/previous lesson buttons)
Comments/Discussion section (optional but highly beneficial)

<img width="1440" alt="video block template from udemy" src="https://github.com/shadcn-ui/ui/assets/36007729/21faf90b-3785-42d6-88b7-4b2e381d8185">

**Comment by himanshu62045** [comment]
+1

**Comment by adi0193** [comment]
+1
